### PR TITLE
[FIX] Fix to get purchase suggest work without the need to patch

### PR DESCRIPTION
**Pull request by levkar**
There's no need to make any patch after this fix.

### CODE REVIEW

File: purchase_suggest/stock.py [hunk 1]
```diff
@@ -20,10 +20,20 @@
...
+
+
+class ProcurementOrder(models.Model):
+    _inherit = 'procurement.order'
+
+    @api.model
+    def _product_virtual_get(self, order_point):
+        if order_point.suggest:
+            return None
+        return (ProcurementOrder, self)._product_virtual_get(order_point)
```
Comment (alexis-via) [inline purchase_suggest/stock.py]: We should call super() here, shouldn't we ?

**Review by alexis-via — CHANGES_REQUESTED** [review]
Thanks for this great contribution !

File: purchase_suggest/stock.py (hunk 1 above)
Comment (levkar) [reply purchase_suggest/stock.py]: Yes I fixed it :)

**Comment by alexis-via** [comment]
Thanks for the update. I'll take the time to test it in the coming days and then I'll merge your PR.

**Pull request merged**